File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_RP_DSOperator.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_RP_DSOperator } from './NS_RP_DSOperator.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_RP_DSOperatorService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Ma', 'OperatorCode', 'OperatorName', 'Type'];

    List: NS_RP_DSOperator[] | undefined;
    ListFilter: NS_RP_DSOperator[] | undefined;
    FormData!: NS_RP_DSOperator;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_RP_DSOperator";
    }
}

File: Service_eHospital_DongNai_A_NSTL/Implement/NS_RP_DSOperatorService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_RP_DSOperatorService : BaseService<NS_RP_DSOperator, INS_RP_DSOperatorRepository>
    , INS_RP_DSOperatorService
    {
    private readonly INS_RP_DSOperatorRepository _NS_RP_DSOperatorRepository;
    public NS_RP_DSOperatorService(INS_RP_DSOperatorRepository NS_RP_DSOperatorRepository) : base(NS_RP_DSOperatorRepository)
    {
    _NS_RP_DSOperatorRepository = NS_RP_DSOperatorRepository;
    }
    }
    }

